script.js

```diff
@@ -0,0 +1,65 @@
+function invokeRZP() {
+    var data = razorpay_script_vars.data;
+
+    var setDisabled = function(id, state) {
+        if (typeof state === "undefined") {
+            state = true;
+        }
+
+        var elem = document.getElementById(id);
+
+        if (state === false) {
+            elem.removeAttribute("disabled");
+        } else {
+            elem.setAttribute("disabled", state);
+        }
+    };
+
+    // Payment was closed without handler getting called
+    data.modal = {
+        ondismiss: function() {
+            setDisabled("btn-razorpay", false);
```

Comment:
I'd much rather have a way to retry it easily, like how we still show the Pay button in woocommerce